### DISCUSSION TITLE
可以精简或者合并实体么？

### DISCUSSION THREAD

**Discussion by gujiangjiang** [XiaoMi/ha_xiaomi_home#241] [Ideas]
我也算资深米家用户了，家里100多个米家设备，之前用miot的时候还算好，实体几百个不算多，这次换成官方的插件后，同步出来1000多个实体，整个页面加载缓慢，无法正常使用了。。。

![IMG_3323](https://github.com/user-attachments/assets/9f78e06c-aa82-4791-bab8-5945f52b6182)

**Comment by JimmyKmi** [comment]
首页的实体可以按需加载，集成会支持所有spec实体，官方也无法区分哪些实体“是有用的”。你可以考虑禁用无必要实体，或者不显示非标准实体。

**Comment by SusanPhevos** [comment]
`custom_components/xiaomi_home/miot/specs/spec_filter.json`能禁止MIoT-Spec-V2实例生成实体，详见[这里](https://github.com/XiaoMi/ha_xiaomi_home/blob/main/doc/README_zh.md#miot-spec-v2-%E8%BF%87%E6%BB%A4%E8%A7%84%E5%88%99)
